component/input/input.css

```diff
@@ -0,0 +1,44 @@
+.input-box{
+    position: relative;
+    background-color: transparent;
+    border-radius: 5px;
+    padding: 6px 6px 6px 6px;
```

Comment:
you can just give padding: 6px; here

Reply:
oh I missed it! Thanks for letting me know